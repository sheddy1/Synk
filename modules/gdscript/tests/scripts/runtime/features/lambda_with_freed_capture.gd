# https://github.com/godotengine/godot/issues/79707

func test():
	var node := Node.new()
	var lambda = func(): print(node)
	node.free()
	lambda.call()
