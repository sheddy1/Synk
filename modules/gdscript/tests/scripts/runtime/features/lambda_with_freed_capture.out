GDTEST_OK
<Freed Object>
